hadoop-tools/hadoop-distcp/src/site/markdown/DistCp.md.vm

```diff
@@ -362,6 +362,7 @@ Command Line Options
 | `-copybuffersize <copybuffersize>` | Size of the copy buffer to use. By default, `<copybuffersize>` is set to 8192B | |
 | `-xtrack <path>` | Save information about missing source files to the specified path. | This option is only valid with `-update` option. This is an experimental property and it cannot be used with `-atomic` option. |
 | `-direct` | Write directly to destination paths | Useful for avoiding potentially very expensive temporary file rename operations when the destination is an object store |
+| `-useIterator` | Uses single threaded listStatusIterator to build listing | Useful for saving memory at the client side. |
```

Comment:
Does it implicitly disable -numListstatusThreads? sounds like a bad new for running distcp on cloud storage where latency is big.

Reply:
Thanx @jojochuang for having a look.
Yes, It indeed isn't meant for object stores, I am trying a multi threaded approach for object stores too as part of HADOOP-17558, that won't be too much memory efficient, but still find a balance between speed and memory. I have a WIP patch for that as well, will share that on the jira

This is basically for HDFS or FS where listing is not slow, but there are memory constraints, my scenario is basically for DR, where it is in general HDFS->HDFS or HDFS->S3

Reply:
I was think we shouuld update the doc to mention it will disable -numListstatusThreads. But if we can merge that WIP patch soon then it's fine.

Reply:
I have updated the document, Let me know if something more can be improved.